template_feed/Microsoft.DotNet.Web.ProjectTemplates.1.x/content/StarterWeb-CSharp/.template.config/template.json

```diff
@@ -157,7 +157,11 @@
     },
     "HttpsPortGenerated": {
       "type": "generated",
-      "generator": "port"
+      "generator": "port",
+      "parameters": {
+        "low": 44300,
+        "high": 44400
```

Comment:
I should have mentioned this in the other PR, but it didn't sink in until I saw the configuration in the json.